kolibri/plugins/facility_management/assets/src/views/user-page/edit-user-modal.vue

```diff
@@ -79,6 +79,8 @@
   export default {
     name: 'editUserModal',
     $trs: {
+      nameMaxLength: '{nameMaxLength, number, integer}',
```

Comment:
As the strings here have no content apart from the number, they are not necessary, instead you can directly invoke `$formatNumber` rather than `$tr`: https://www.npmjs.com/package/vue-intl#formatnumber

Reply:
Neat, didn't know that existed!

Note that this particular string is another instance of the [comment above](maxlength) and needs no localization at all.

Reply:
I have tried to enforce it, but evidently it has not caught on :)

https://github.com/learningequality/kolibri/search?utf8=%E2%9C%93&q=%24formatNumber&type=

Reply:
@rtibbles & @indirectlylit  I already used the `$formatNumber` and it worked. The only task remaining is how I can localised the [valueLength](https://github.com/learningequality/Keen-UI/blob/17911401dff3d65f78bc1edbc32acf675864da03/src/UiTextbox.vue#L84)

Reply:
@mrpau-richard please read the comments from me and @benjaoming again, and ping me in slack if you have questions.

The `maxlength` property of text boxes is _not_ a user-facing string and should _not_ be localized.

Reply:
> The only task remaining is how I can localised the valueLength

I created a new issue for this here: #3141

That one will be a little more complicated because we may need to fork the keen textbox component. (cc @christianmemije)